src/components/modal/LoginCardModal.jsx

```diff
@@ -85,7 +103,7 @@ const LoginCardModal = () => {
 					/>
 					<InputField
 						labelText='Password'
-						type='password'
+						type={!showPassword ? 'password' : 'text'}
```

Comment:
Please maintain one more field in the state object to hide and show the confirm password field. 
Please enable hide and show password on login.

Reply:
https://github.com/zoef-yt/react-ecom/pull/20/commits/98b8250aa90e0726b24de252cbcad7a704c54622